sdcm/utils/sstable/sstable_utils.py

```diff
@@ -51,24 +54,38 @@ def get_sstables(self, from_minutes_ago: int = 0):
         self.log.debug('Got %s sstables %s', len(selected_sstables), message)
         return selected_sstables
 
-    def is_sstable_encrypted(self, sstables=None) -> list:
+    def is_sstable_encrypted(self, sstables=None) -> list:  # pylint: disable=too-many-branches
         if not sstables:
             sstables = self.get_sstables()
         if isinstance(sstables, str):
             sstables = [sstables]
         sstables_encrypted_mapping = {}
-        for sstable in sstables:
-            sstables_res = self.db_node.remoter.sudo(
+        if ComparableScyllaVersion(self.db_node.scylla_version) >= '2023.2':
```

Comment:
Shouldn't the version have the `rc` part?

Reply:
It should...

It's quite an annoying caveat, and I'm guessing it's not the last time we're gonna miss it

Reply:
GA versions are always newer than the `RC` ones.
So, the behavior is correct.

Reply:
changed